cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -450,8 +447,8 @@ def _validate_feature_id(self, feature_id: str, df_name: str):
         return
 
     @staticmethod
-    def _chunk_matrix(
-        matrix: Union[np.ndarray, sparse.spmatrix],
+    def chunk_matrix(
+        matrix: Union[np.ndarray, sparse.spmatrix, DaskArray],
```

Comment:
Are there any cases where we use `chunk_matrix` that isn't a DaskArray. I think `read_backed` does a good job of converting that uses it a dask array. If it doesn't, then that element likely should be read into a dask array.

TLD: I think we can remove support for `np.ndarray, sparse.spmatrix,` in `chunk_matrix`

Reply:
We can even remove `chunk_matrix` all together if we don't need to support the other data types with this function. Dask is already doing the chunking. That work could be apart of a following PR to improve processing performance.